pubspec.yaml

```diff
@@ -126,7 +126,7 @@ dependencies:
   in_app_purchase: ^3.2.1
 
   # Ads
-  google_mobile_ads: ^6.0.0
+  google_mobile_ads: ^5.3.1
```

Comment:
I had to downgrade a few dependencies to build Android and preserve API Level 21 support for now. Will fix this in the next release